Meadowlark-js/backends/meadowlark-mongodb-backend/test/integration/authorization/GetAllAuthorizationClients.test.ts

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { CreateAuthorizationClientRequest } from '@edfi/meadowlark-authz-server';
+import { MongoClient } from 'mongodb';
+import { getAuthorizationCollection, getNewClient } from '../../../src/repository/Db';
+import { createAuthorizationClientDocument } from '../../../src/repository/authorization/CreateAuthorizationClient';
+import { getAllAuthorizationClientDocuments } from '../../../src/repository/authorization/GetAllAuthorizationClients';
+
+jest.setTimeout(40000);
```

Comment:
Given this is defined in multiple test files, we could add a global default timeout by using [setupFilesAfterEnv](https://jestjs.io/docs/configuration#setupfilesafterenv-array). This is also done before each test, but we avoid having to repeat this per test.
(This should not be part of this ticket, just wanted to point it out)

Reply:
@bradbanister what are your thoughts about that?